app/src/hooks/cb_api/useLookupApi.tsx

```diff
@@ -1,5 +1,7 @@
 import { AxiosInstance } from 'axios';
 
+export type OrderBy = 'asc' | 'desc';
```

Comment:
Is this is already export/provided by a more relevant file?

Reply:
There is GridSortDirection which I could use. It has some additional types I could strip off

Reply:
Honestly you're welcome to do either or on this one :+1: